3CodeChallenge/app/app/src/main/res/layout/views_sample_challenge2.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        android:padding="@dimen/small_padding">
+
+        <RelativeLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/yellowButton"
+                android:layout_width="64dp"
+                android:layout_height="64dp"
+                android:background="@color/yellow" />
+
+            <TextView
+                android:id="@+id/lightBlueButton"
+                android:layout_width="64dp"
```

Comment:
Be consistent and add all the hardcoded values in dimen.xml. Also please be careful and don't set hardcoded values for width and height, especially for TextView. This approach could be applied sometimes for ImageView.